mod error_handling;

fn main() {
    error_handling::run();
}
